app/src/main/java/com/babylon/wallet/android/presentation/account/AccountViewModel.kt

```diff
@@ -57,7 +57,7 @@ class AccountViewModel @Inject constructor(
 
         viewModelScope.launch {
             appEventBus.events.filter { event ->
-                event is AppEvent.GotFreeXrd || event is AppEvent.Status.Transaction.Success || event is AppEvent.RestoredMnemonic
```

Comment:
Yes, if we rely on transaction success, and we dismiss the dialog, accounts would not refresh. Polling for transaction was detached from the UI before this change, and it is still detached. Because of that I introduced separate event which is sent from that polling code in app scope, instead of relying on success event that is triggered from the UI (if it is not dismissed)